app.json

```diff
@@ -4,10 +4,10 @@
   },
   "env": {
     "ADEPT_SCALE_LICENSE_KEY": {
-      "required": true
+      "required": false
```

Comment:
you should just remove these env vars because they aren't actually a part of staging.